arch/risc-v/src/k230/k230_hart.c

```diff
@@ -158,7 +151,8 @@ void k230_hart_init(void)
 
 bool k230_hart_is_big(void)
 {
-  return g_big;
+  sinfo("g_misa=%lx\n", g_misa);
```

Comment:
should we remove the log or move after line 144

Reply:
The line 144 runs very earlier in M-mode, thus `sinfo` is not ready yet. 
This log means to show the result got from S-mode later.